lesson02/assignment/index.html

```diff
@@ -7,8 +8,28 @@
         <title>Assignment 02</title>
     </head>
     <body>
+        <!-- Outter gray box-->
         <main class="main">
-            <h1>title</h1>
+          <!-- White box-->
+          <div class="background">
+            <!--Container for photo/name/title-->
+            <div class="photo-title">
+              <img src="img/unsplash-headshot.jpg" class="photo-title__headshot" alt="Headshot for Steve Smith in blue shirt">
+              <h1 class="photo-title__name">Steve Smith</h1>
+              <h2 class="photo-title__title">Project Manager</h2>
+            </div>
+            <!-- Container for info box-->
+            <div class="info">
+              <div class="profile">
+                <p class="profile__para"><strong>Company:</strong> Front End Dev Co</p>
+                <p class="profile__para"><strong>Experience:</strong> 3 years</p>
+                <p class="profile__para"><strong>School:</strong> UW</p>
+                <p class="profile__para"><strong>Major:</strong> Marketing</p>
+                <p class="profile__para"><strong>Email:</strong> steve@fedc.com</p>
+                <p class="profile__para"><img src="img/linkedin.svg" class="linkedin" alt="LinkedIn logo"> steve.linkedinprofile.com</p>
```

Comment:
I'm not seeing any styles applied for this class. Maybe it's not necessary anymore? Also, if a style needs to be applied to every `p` element within a given parent, it's simpler to select on the parent. In this case, that might look like `.profile > p`. This isn't a hard and fast rule, but should probably be the default unless you have a strong reason to select each child individually.